### ISSUE TITLE
dependency on cofh lib

### ISSUE THREAD

**Issue by RubilaxXxx** [AirBurn0/am2playground#27]
you're using cofh lib math helper in EntitySpellMissile causing crash when the mod isn't there 
[crash-2024-10-21_23.08.23-client.txt](https://github.com/user-attachments/files/17466980/crash-2024-10-21_23.08.23-client.txt)

**Comment by AirBurn0** [comment]
> you're using cofh lib math helper in EntitySpellMissing causing crash when the mod isn't there [crash-2024-10-21_23.08.23-client.txt](https://github.com/user-attachments/files/17466980/crash-2024-10-21_23.08.23-client.txt)

Yep thats really stupid bug that I fixed in [dev versions](https://github.com/AirBurn0/am2playground/releases/tag/v0.11.4-beta5) but that's still the case for release version until I got myself to finish update
